extra/index.json

```diff
@@ -1317,6 +1317,38 @@
 			},
 			"icon": "ted",
 			"type": "charter"
+		},
+		{
+			"ids": [ "santarockstar", "santarockstarhd", "santa" ],
+			"names": {
+				"en-US": "Santa Rockstar"
+			},
+			"icon": "santahd",
+			"type": "game"
+		},
+		{
+			"ids": [ "Mickelräven", "Mickelraven" ],
```

Comment:
Source ids should always be lowercase. The game will (or rather, should) convert it into lowercase for you.